docs/pages/guides/editing-richtext.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Editing rich text
+description: Learn about current approaches to previewing and editing rich text in react-native.
+---
+
+A lot of applications need to allow users to type in text. For example, if you want to build a messaging or social media app, you will probably rely heavily on text inputs. React Native has a built-in `<TextInput>` component to implement this without much effort for many simple cases.
+
+Sometimes however, you need to be more flexible. Think of long social media posts, note apps or document editors. Ideally you need to allow different text styles, lists, headings, embedded images and more. This is called a rich text editor, and it’s a difficult problem to solve everywhere, including in React Native.
+
+There’s currently no obvious, default solution for that in the React Native ecosystem, but we can explore some options and promising approaches, each with their own tradeoffs.
+
+# Rendering rich text
+
+If you don’t need to actually edit rich text, just display it, there are a lot of great options:
+
+- If you have markdown content, you can use a markdown renderer like [react-native-markdown-renderer](https://www.npmjs.com/package/react-native-markdown-renderer) or other.
+
+- If you have HTML content, you can use [@expo/html-elements](https://www.npmjs.com/package/@expo/html-elements) or simply a webview ([react-native-webview](https://docs.expo.dev/versions/latest/sdk/webview/)).
+
+- If you have a custom format and want more control, you can take advantage of nesting `<Text>` components to render styles and layouts.
+
+  ```jsx
+  <TextInput>
+    <Text>
+      <Text style={{ fontWeight: 900 }}>Some bold text</Text>Some regular text
+    </Text>
+  </TextInput>
+  ```
+
+- You can also use [Expo Modules](/modules/overview/) to write a custom renderer component with native platform primitives such as `AttributedString` on iOS and 3rd party libraries such as [Markwon](https://github.com/noties/Markwon) on Android.
```

Comment:
```suggestion
- You can also use [Expo Modules](/modules/overview/) to write a custom renderer component with native platform primitives using third-party libraries such as [Markwon](https://github.com/noties/Markwon) on Android and `AttributedString` on iOS.
```

[Readjusted platform order as per our writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web)